doc/classes/Rect2.xml

```diff
@@ -50,63 +50,83 @@
 			<param index="2" name="width" type="float" />
 			<param index="3" name="height" type="float" />
 			<description>
-				Constructs a [Rect2] by x, y, width, and height.
+				Constructs a [Rect2] by setting its [member position] to [param x] and [param y], and its [member size] to [param width] and [param height].
 			</description>
 		</constructor>
 	</constructors>
 	<methods>
 		<method name="abs" qualifiers="const">
 			<return type="Rect2" />
 			<description>
-				Returns a [Rect2] with equivalent position and area, modified so that the top-left corner is the origin and [code]width[/code] and [code]height[/code] are positive.
+				Returns a [Rect2] equivalent to this rectangle, with its [member position] shifted to be the top-left corner, and its [member size] set to be a positive value.
```

Comment:
There has to be a better way to word this. The reason to use this method is to get rid of negative sizes, the position adjustment is a side effect to keep the rectangle identical. Besides, saying "negative size" and the likes is awkward to begin with, it's a vector.

Reply:
Mentioning Rect2's `size` becoming an absolute value first, then the `position` shift could be a start. Alternatively, the two corners (`position` and `end`) could be mentioned, instead.

Reply:
I like the rewording! Truer words were never spoken.